src/pudl/io_managers.py

```diff
@@ -510,6 +557,13 @@ def load_input(self, context: InputContext) -> pd.DataFrame:
                 "the tables that does not get loaded into the PUDL SQLite DB because "
                 "it's a work in progress or is distributed in Apache Parquet format."
             )
+        if self.settings.read_from_parquet:
+            logger.info(f"Reading table {self.db_name}/{table_name} from parquet.")
+            schema = res.to_pyarrow()
+            return pq.read_table(
+                source=PudlPaths().parquet_path(self.db_name, table_name),
+                schema=schema,
+            ).to_pandas()
```

Comment:
I don't think this pandas -> parquet -> pandas round-trip will necessarily preserve dtypes, since we're not using pyarrow dtypes in our dataframes, so I suspect we'll still need to`res.enforce_schema()` here as we do when reading out of SQLite below to avoid unexpected dtype problems downstream.